x-pack/plugins/observability/public/pages/slo_edit/constants.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { CreateSLOParams } from '@kbn/slo-schema';
+
+import {
+  BUDGETING_METHOD_OPTIONS,
+  TIMEWINDOW_OPTIONS,
+} from './components/slo_edit_form_objectives';
+
+export const SLI_OPTIONS = [
+  {
+    value: 'sli.kql.custom' as const,
+    text: i18n.translate('xpack.observability.slos.sloTypes.kqlCustomIndicator', {
+      defaultMessage: 'KQL custom indicator',
+    }),
+  },
+];
+
+export const SLO_EDIT_FORM_DEFAULT_VALUES: CreateSLOParams = {
+  name: '',
+  description: '',
+  indicator: {
+    type: SLI_OPTIONS[0].value,
+    params: {
+      index: '',
+      filter: '',
+      good: '',
+      total: '',
+    },
+  },
+  timeWindow: {
+    duration: TIMEWINDOW_OPTIONS[0].value as any, // Get this to be a proper Duration
```

Comment:
Technically, this is simply a `string` that needs to follow a special format: `{duration}{unit}`. But from a type point of view, it's a string.

Reply:
I need to cast as `any` here, otherwise I'll get an error. I could do `as unknown as string`, perhaps that is more correct than just any